yb-voyager/src/srcdb/postgres.go

```diff
@@ -469,3 +469,32 @@ func generateSSLQueryStringIfNotExists(s *Source) string {
 func (pg *PostgreSQL) GetServers() []string {
 	return []string{pg.source.Host}
 }
+
+func (pg *PostgreSQL) GetChildPartitions(tableName *sqlname.SourceName) []*sqlname.SourceName {
```

Comment:
refactoring opportunity: (now or later): since both of the postgres/yugabyte structs share a lot of code, we can consider reusing the common code somehow (maybe separating out the logic into separate functions? Maybe yugabyte embeds postgres?)

Reply:
Yes, I think we discussed this earlier as well, we should do this but I think later is better because that will need a lot of code changes and testing for synchronize cmd, etc. I can add it to tech debt and we can pick this up after the release ?

Reply:
yep yep, pls create a jira 👍

Reply:
https://yugabyte.atlassian.net/browse/DB-8375